rubrics/api/src/main/bundle/rubrics.properties

```diff
@@ -64,7 +64,7 @@ search_rubrics=Search Rubrics by title, site or author:
 percent_sign=%
 draft_evaluation=Draft rubric evaluation. {}
 draft_evaluation_sakai.gradebookng=Save the grade to publish the rubric to the student.
-draft_evaluation_sakai.assignment=Save and release to student to publish the rubric to the student.
+draft_evaluation_sakai.assignment.grades=Save and release to student to publish the rubric to the student.
```

Comment:
Please update the properties for the other languages, otherwise they'll be orphaned and it will result in future bugs.

Reply:
Good point, thanks.